sentences/sv/lock_HassTurnOff.yaml

```diff
@@ -0,0 +1,17 @@
+language: sv
+intents:
+  HassTurnOff:
+    data:
+      - sentences:
+          - "lås upp <name>[ i_på <area>]"
+        requires_context:
+          domain: lock
+        response: lock
+
+      - sentences:
+          - "lås upp [<alla>] <låsbar> <i_på> <area>"
+          - "lås upp [<alla>] <area> <låsbar>"
+        slots:
+          domain: "lock"
+          name: "alla"
```

Comment:
I would expect that this sentence should set the name to "all" and not "alla". Since it is not translated slot name.
Looking at DE and ES languages for example.

Reply:
Oh. My bad. Fixed in latest commit.